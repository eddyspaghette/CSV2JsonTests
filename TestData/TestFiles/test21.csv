1!100!200!300!400!500
21!120!220!320!420!520
22!121!221!321!421!521
6!105!205!!405!505
7!106!206!306!406!506
8!107!207!!407!507
9!108!208!308!408!508
10!109!209!309!409!509
11!110!210!310!410!510
12!!!311!411!511
13!112!212!312!412!512
14!!213!313!413!513
15!114!214!314!414!514
16!115!215!315!415!515
17!116!216!!416!516
18!117!217!317!417!517
19!118!218!318!418!518
20!119!219!319!419!519
26!125!225!325!425!525
27!126!226!326!!526
23!122!222!!422!522
24!123!!323!423!523
25!124!224!324!424!524
28!!227!327!427!527
29!128!228!328!428!528
2!101!201!!401!501
3!102!202!302!402!502
4!103!203!303!403!
5!104!204!304!404!504
