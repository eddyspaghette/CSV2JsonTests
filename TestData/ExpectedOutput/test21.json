[
 {
   "FIELD1": "1!100!200!300!400!500"
 },
 {
   "FIELD1": "21!120!220!320!420!520"
 },
 {
   "FIELD1": "22!121!221!321!421!521"
 },
 {
   "FIELD1": "6!105!205!!405!505"
 },
 {
   "FIELD1": "7!106!206!306!406!506"
 },
 {
   "FIELD1": "8!107!207!!407!507"
 },
 {
   "FIELD1": "9!108!208!308!408!508"
 },
 {
   "FIELD1": "10!109!209!309!409!509"
 },
 {
   "FIELD1": "11!110!210!310!410!510"
 },
 {
   "FIELD1": "12!!!311!411!511"
 },
 {
   "FIELD1": "13!112!212!312!412!512"
 },
 {
   "FIELD1": "14!!213!313!413!513"
 },
 {
   "FIELD1": "15!114!214!314!414!514"
 },
 {
   "FIELD1": "16!115!215!315!415!515"
 },
 {
   "FIELD1": "17!116!216!!416!516"
 },
 {
   "FIELD1": "18!117!217!317!417!517"
 },
 {
   "FIELD1": "19!118!218!318!418!518"
 },
 {
   "FIELD1": "20!119!219!319!419!519"
 },
 {
   "FIELD1": "26!125!225!325!425!525"
 },
 {
   "FIELD1": "27!126!226!326!!526"
 },
 {
   "FIELD1": "23!122!222!!422!522"
 },
 {
   "FIELD1": "24!123!!323!423!523"
 },
 {
   "FIELD1": "25!124!224!324!424!524"
 },
 {
   "FIELD1": "28!!227!327!427!527"
 },
 {
   "FIELD1": "29!128!228!328!428!528"
 },
 {
   "FIELD1": "2!101!201!!401!501"
 },
 {
   "FIELD1": "3!102!202!302!402!502"
 },
 {
   "FIELD1": "4!103!203!303!403!"
 },
 {
   "FIELD1": "5!104!204!304!404!504"
 }
]